.circleci/config.yml

```diff
@@ -68,22 +68,32 @@ jobs:
         name: create ilastik conda environment
         command: >
             cd ${ILASTIK_ROOT}/ilastik-meta &&
-            mamba env create --name ${TEST_ENV_NAME} --file ilastik/dev/environment-dev.yml &&
-            mamba install --name ${TEST_ENV_NAME} --freeze-installed -c ilastik-forge -c conda-forge mpi4py pytest-cov volumina &&
-            mamba run -n ${TEST_ENV_NAME} pip install -e ilastik
+            conda env create --name ${TEST_ENV_NAME} --file ilastik/dev/environment-dev.yml &&
+            conda install --name ${TEST_ENV_NAME} --freeze-installed -c ilastik-forge -c conda-forge mpi4py pytest-cov volumina &&
+            conda run -n ${TEST_ENV_NAME} pip install -e ilastik
 
+    - run:
+        name: start xvfb
+        command: >
+          Xvfb :42 -screen 0 1024x768x24
+        background: true
+    - run:
+        name: wait for xvfb to be ready
+        command: while [ ! -e /tmp/.X11-unix/X42 ]; do sleep 0.1; done
     - run:
         name: run ilastik tests
+        environment:
+            DISPLAY: :42
         command: >
             cd ${ILASTIK_ROOT}/ilastik-meta/ilastik &&
             mkdir test-results &&
-            xvfb-run --server-args="-screen 0 1024x768x24" mamba run -n ${TEST_ENV_NAME} pytest --cov-report=xml --cov=lazyflow --cov=ilastik --run-legacy-gui --junitxml=test-results/junit.xml &&
+            conda run -n ${TEST_ENV_NAME} pytest -v --run-legacy-gui --cov-report=xml --cov=lazyflow --cov=ilastik --junitxml=test-results/junit.xml &&
```

Comment:
Is the leftover `-v` intentional here? Weirdly, I can't actually see a difference in verbosity between the new and some old logs... 🤷

Reply:
I'd want to keep -v as the it's easier to follow test progress.